astro/src/content/docs/lifecycle/migrate-users/index.mdx

```diff
@@ -1,27 +1,83 @@
 
 ## Overview
 
 Oftentimes you are migrating to FusionAuth from another solution, whether it be homegrown or another vendor.
 
 These guides and resources will help you understand and scope the migration process.
 
-## Migration Guides
+There are multiple methods you can use to migrate users into FusionAuth.
+
+* Bulk migration
+* Connectors
+* SCIM
+* The User API
+
+The FusionAuth [migration guide](/docs/lifecycle/migrate-users/bulk/general) has general migration guidance, strategies and help.
+
+## Bulk Migration
+
+Bulk migration lets you import users at one point in time, making for a clean cutover. You can import password hashes, including hashes performed with a custom, non-standard algorithm, making a migration seamless to your users. You use the [User Import API](/docs/apis/users#import-users) to perform this. You can also [import refresh tokens](/docs/apis/users#import-refresh-tokens), which helps if you want your users to be able to refresh access tokens transparently.
+
+A bulk migration is a good choice when you:
+
+* want to migrate users all at once
+* are migrating into a new FusionAuth instance
+* have access to password hashes from a previous auth system
+
+### Provider Specific Bulk Migration Guides
 
-* [General](/docs/lifecycle/migrate-users/bulk/general) - general migration guidance, strategies and help
 * [Auth0](/docs/lifecycle/migrate-users/bulk/auth0) - how to migrate from Auth0 to FusionAuth
 * [Microsoft Azure AD B2C](/docs/lifecycle/migrate-users/bulk/azureadb2c) - how to migrate from Microsoft Azure AD B2C to FusionAuth
 * [Cognito](/docs/lifecycle/migrate-users/bulk/cognito) - how to migrate from Amazon Cognito to FusionAuth
 * [Duende IdentityServer](/docs/lifecycle/migrate-users/bulk/duende) - how to migrate from Duende IdentityServer to FusionAuth
 * [Firebase](/docs/lifecycle/migrate-users/bulk/firebase) - how to migrate from Firebase to FusionAuth
 * [Keycloak](/docs/lifecycle/migrate-users/bulk/keycloak) - how to migrate from Keycloak to FusionAuth
 * [Supabase](/docs/lifecycle/migrate-users/bulk/supabase) - how to migrate from Supabase to FusionAuth
-* [Tutorial](/docs/lifecycle/migrate-users/bulk/tutorial) - how to migrate from a single user database to FusionAuth
+* [Tutorial](/docs/lifecycle/migrate-users/bulk/tutorial) - how to migrate from an example homegrown user database to FusionAuth
+
+## Connectors
+
+<PremiumEditionBlurb />
+
+Connectors allow you to migrate users one by one transparently. This is called a slow or drip migration, because each user is migrated at the time of login. 
+
+A slow migration is a good choice when you:
+
+* are migrating from a system which doesn't allow access to password hashes
+* are migrating users into a FusionAuth instance with existing users
+
+Learn [more about slow migrations](/articles/identity-basics/slow-migration) or [how to use Connectors with FusionAuth](/docs/lifecycle/migrate-users/connectors).
+
+## SCIM
+
+<EnterpriseEditionBlurb />
+
```

Comment:
this is cool, I did not realize SCIM fell into migration

Reply:
Neither did I until I wrote this :) . It seems like a migrationish solution to me, because you could use it once and then turn it off.